pokecli.py

```diff
@@ -104,6 +108,34 @@ def report_summary(bot):
     if metrics.most_perfect is not None:
         logger.log('Most Perfect Pokemon: {}'.format(metrics.most_perfect['desc']), 'cyan')
 
+def snipe():
```

Comment:
This shouldn't be here. Now that the bot is fully configurable by the user in config.json, I'd recommend creating a SnipePokemon worker. Then a user could specify just that in their config.json and get the behavior you want. There should generally be no reason this file needs to be modified anymore.